src/main/java/me/jellysquid/mods/sodium/client/SodiumClientMod.java

```diff
@@ -17,6 +21,18 @@ public void onInitializeClient() {
 
     }
 
+    public static String version() {
+        if (VERSION == null) {
+            ModContainer ourContainer = FabricLoader.getInstance().getModContainer("sodium")
+                    .orElseThrow(() -> new IllegalStateException("Mod with ID \"sodium\" couldn't be found... but that's us!"));
+            VERSION = ourContainer.getMetadata().getVersion().getFriendlyString();
+            if ("${version}".equals(VERSION))
+                VERSION = Formatting.RED + "(!!) Devel";
```

Comment:
For clarity, and in order to match the style elsewhere in Sodium, this should be either on the same line as the if:

```java
            if ("${version}".equals(VERSION)) VERSION = Formatting.RED + "(!!) Devel";
```

Or wrapped in braces:

```java
            if ("${version}".equals(VERSION)) {
                VERSION = Formatting.RED + "(!!) Devel";
            }
```

I would recommend the latter.